packages/react/src/recipes/FlexTemplate/FlexSection/FlexSection.stories.tsx

```diff
@@ -0,0 +1,766 @@
+          },
+        }
+      : null,
+    breakoutBanner: args.showBreakoutBanner
+      ? {
+          fields: {
+            align: args.breakoutBannerAlign,
+            showLogo: args.breakoutBannerShowLogo,
+            headingLevel: args.breakoutBannerHeadingLevel,
+            showFootnotes: args.breakoutBannerShowFootnotes,
+          },
+        }
+      : null,
+
+    statistics: args.showStatistics
+      ? {
+          fields: {
+            count: args.statisticsCount,
+            variant: args.statisticsVariant,
+            size: args.statisticsSize,
+            showDescription: args.statisticsShowDescription,
+            descriptionVariant: args.statisticsDescriptionVariant,
+            showDescriptionFootnotes: args.statisticsShowDescriptionFootnotes,
+          },
+        }
+      : null,
+    pricingOptions: args.showPricingOptions
+      ? {
+          fields: {
+            variant: args.pricingOptionsVariant,
+            align: args.pricingOptionsAlign,
+            showFootnotes: args.pricingOptionsShowFootnotes,
+            showFeatureList: args.pricingOptionsShowFeatureList,
+            headingLevel: args.pricingOptionsHeadingLevel,
+          },
+        }
+      : null,
+    segmentedControlPanel: args.showSegmentedControlPanel
+      ? {
+          fields: {},
+        }
+      : null,
+    testimonials: args.showTestimonials
+      ? {
+          fields: {
+            testimonialCount: args.testimonialCount,
+            backgroundImageVariant: args.testimonialBackgroundImageVariant,
+            variant: args.testimonialVariant,
+            displayedAuthorImage: args.testimonialDisplayedAuthorImage,
+            quoteMarkColor: args.testimonialQuoteMarkColor,
+          },
+        }
+      : null,
+    rivers: args.showRivers
+      ? {
+          fields: {
+            type: args.riverType,
+            align: args.riverAlign,
+            ctaVariant: args.riverCtaVariant,
+            hasCta: args.riverHasCta,
+            hasLeadingVisual: args.riverHasLeadingVisual,
+            hasShadow: args.riverHasShadow,
+            hasTrailingComponent: args.riverHasTrailingComponent,
+            imageTextRatio: args.riverImageTextRatio,
+            label: args.riverHasLabel,
+            labelColor: args.riverLabelColor,
+            labelSize: args.riverLabelSize,
+            visualType: args.riverVisualType,
+          },
+        }
+      : null,
+    visualSettings: {
+      fields: {
+        backgroundColor: args.backgroundColor,
+        paddingBlockStart: args.paddingBlockStart,
+        colorMode: args.colorMode,
+        paddingBlockEnd: args.paddingBlockEnd,
+        roundedCorners: args.roundedCorners,
+        verticalGap: args.verticalGap,
+        hasBorderBottom: args.hasBorderBottom,
+        enableRiverStoryScroll: args.enableRiverStoryScroll,
+      },
+    },
+  },
+})
+
+const meta: Meta<FlexSectionStoryArgs> = {
+  title: 'Recipes/FlexTemplate/FlexSection',
```

Comment:
Could we also get stories for FlexTemplate, which is just flex section + a hero so we start seeing this flexsection in practice?

E.g. Could we have a story here: recipes/flextemplate/examples/models

Which reproduces this page https://github.com/features/models, which I believe was built on flex template and should look like-for-like if flexsection is correctly implemented.